src/desktop/apps/article/components/InfiniteScrollNewsArticle.js

```diff
@@ -0,0 +1,167 @@
+import * as React from 'react'
```

Comment:
Now that Force supports TypeScript, can you refactor this following TS patterns?